lib/resources/r4/questionnaire_response.yaml

```diff
@@ -23,4 +23,168 @@ fields:
     example: |
       {
         "id": "SH-12508041"
-      }
\ No newline at end of file
+      }
+
+  - name: questionnaire
+    required: 'Yes'
+    type: Questionnaire
+    url: http://hl7.org/fhir/r4/questionnaireresponse-definitions.html#QuestionnaireResponse.questionnaire
```

Comment:
https on all the links

Reply:
https://github.com/cerner/fhir.cerner.com/pull/669/commits/c1998643fdbcdb4398f84aebc9e10ee9cdc5f28c